src/test/java/teammates/ui/webapi/FeedbackSessionClosingRemindersActionTest.java

```diff
@@ -103,14 +105,50 @@ public void testExecute() throws Exception {
         session3.setSentOpenEmail(false); // fsLogic will set the flag to true
         verifyPresentInDatabase(session3);
 
+        // update deadline extensions to have end time within the next 24 hours
+        DeadlineExtensionAttributes deadlineExtensionStudent =
+                typicalBundle.deadlineExtensions.get("student4InCourse1Session1");
+        logic.updateDeadlineExtension(
+                DeadlineExtensionAttributes
+                        .updateOptionsBuilder(
+                            deadlineExtensionStudent.getCourseId(),
+                            deadlineExtensionStudent.getFeedbackSessionName(),
+                            deadlineExtensionStudent.getUserEmail(),
+                            deadlineExtensionStudent.getIsInstructor())
+                        .withEndTime(TimeHelper.getInstantHoursOffsetFromNow(12))
+                        .build());
+        DeadlineExtensionAttributes deadlineExtensionStudentAlreadyAnswered =
+                typicalBundle.deadlineExtensions.get("student3InCourse1Session1");
+        logic.updateDeadlineExtension(
+                DeadlineExtensionAttributes
+                        .updateOptionsBuilder(
+                            deadlineExtensionStudentAlreadyAnswered.getCourseId(),
+                            deadlineExtensionStudentAlreadyAnswered.getFeedbackSessionName(),
+                            deadlineExtensionStudentAlreadyAnswered.getUserEmail(),
+                            deadlineExtensionStudentAlreadyAnswered.getIsInstructor())
+                        .withEndTime(TimeHelper.getInstantHoursOffsetFromNow(12))
+                        .build());
+        DeadlineExtensionAttributes deadlineExtensionInstructor =
+                typicalBundle.deadlineExtensions.get("instructor2InCourse1Session1");
+        logic.updateDeadlineExtension(
+                DeadlineExtensionAttributes
+                        .updateOptionsBuilder(
+                            deadlineExtensionInstructor.getCourseId(),
+                            deadlineExtensionInstructor.getFeedbackSessionName(),
+                            deadlineExtensionInstructor.getUserEmail(),
+                            deadlineExtensionInstructor.getIsInstructor())
+                        .withEndTime(TimeHelper.getInstantHoursOffsetFromNow(12))
+                        .build());
```

Comment:
A bit verbose; can you try to make a list of the users and then use a for loop to update their `endTime`?

```java
DeadlineExtensionAttributes deadlineExtensionStudent =
        typicalBundle.deadlineExtensions.get("student4InCourse1Session1");       
DeadlineExtensionAttributes deadlineExtensionStudentAlreadyAnswered =
        typicalBundle.deadlineExtensions.get("student3InCourse1Session1");
DeadlineExtensionAttributes deadlineExtensionInstructor =
        typicalBundle.deadlineExtensions.get("instructor2InCourse1Session1");
List<DeadlineExtensionAttributes> deadlinesToExtend = List.of(
        deadlineExtensionStudent, 
        deadlineExtensionStudentAlreadyAnswered,
        deadlineExtensionInstructor);
for (var deadlineExtension : deadlinesToExtend) {
    ...
}
```